### ISSUE TITLE
Default overlay alpha is invisible

### ISSUE THREAD

**Issue by mindfilleter** [snoobinoob/lightOverlay#2]
TL;DR; Set the default alpha to something visible, or even 100% opaque. Give some kind of confirmation to the user when they press the toggle key.

When I first tried this mod, I would press the F8 key and nothing would happen. I assumed it was broken due to the new update (which was probably also true prior to #1 ). After pulling DeadMor0z's patch and testing it again I had the same result. I experimented with using the chat command to toggle the overlay and saw the message stating that it had been toggled on, but I still couldn't see it. Finally, I changed the alpha and it showed up.

The defaults should be better and the F8 key should give the same feedback (i.e. chat message) to the user to let them know something is happening. Otherwise the appearance is that the mod isn't working at all.

**Comment by DeadMor0z** [comment]
Done in latest commit.

**Comment by mindfilleter** [comment]
Nice! Thanks Dude. Hope it gets merged soon.

**Comment by DeadMor0z** [comment]
You can download latest build from my fork page.

**Comment by mindfilleter** [comment]
@DeadMor0z Yeah, I have your repo setup as a remote in my local.

**Comment by snoobinoob** [comment]
I have merged https://github.com/snoobinoob/lightOverlay/pull/1 and uploaded the mod to steam workshop. Marking as fixed

**Comment by mindfilleter** [comment]
Woot! Thank you both :)
